aspnetcore/migration/identity.md

```diff
@@ -65,18 +62,21 @@ using Microsoft.Data.Entity;
 
 namespace NewMvc6Project.Models
```

Comment:
Please change this namespace, since MVC 6 is no longer a thing. I suggest using "NewMvcProject" or something similar.